client/my-sites/hosting/sftp-card/index.js

```diff
@@ -94,67 +95,77 @@ const SFTPCard = ( { translate, siteId } ) => {
 					</p>
 				) }
 			</div>
-			{ username && (
-				<table className="sftp-card__info-table">
+			{ ( username || disabled ) && (
+				<table
+					className={ classNames( 'sftp-card__info-table', { [ 'is-placeholder' ]: disabled } ) }
+				>
 					<tbody>
 						{ map( sftpData, ( data, title ) => (
 							<tr key={ title }>
 								<th>{ title }:</th>
 								<td>
-									<span>{ data }</span>
+									<span>{ ! disabled && data }</span>
 								</td>
 							</tr>
 						) ) }
 						<tr>
 							<th>{ translate( 'Username' ) }:</th>
 							<td>
-								<p className="sftp-card__hidden-overflow">{ username }</p>
-								<ClipboardButton
-									text={ username }
-									onCopy={ () => setIsCopied( 'username' ) }
-									compact
-								>
-									{ usernameIsCopied
-										? translate( 'Copied!' )
-										: translate( 'Copy', { context: 'verb' } ) }
-								</ClipboardButton>
-							</td>
-						</tr>
-						<tr>
-							<th>{ translate( 'Password' ) }:</th>
-							<td>
-								{ password ? (
+								{ disabled ? (
+									<span></span>
+								) : (
 									<>
-										<p className="sftp-card__hidden-overflow">{ password }</p>
+										<p className="sftp-card__hidden-overflow">{ username }</p>
 										<ClipboardButton
-											text={ password }
-											onCopy={ () => setIsCopied( 'password' ) }
+											text={ username }
+											onCopy={ () => setIsCopied( 'username' ) }
 											compact
 										>
-											{ passwordIsCopied
+											{ usernameIsCopied
 												? translate( 'Copied!' )
 												: translate( 'Copy', { context: 'verb' } ) }
 										</ClipboardButton>
-										<p className="sftp-card__password-warning">
-											{ translate(
-												"Be sure to save your password somewhere safe. You won't be able to view it again without resetting."
-											) }
-										</p>
-									</>
-								) : (
-									<>
-										<p>{ translate( 'You must reset your password to view it.' ) }</p>
-										<Button
-											onClick={ () => resetAtomicSFTPUserPassword( siteId ) }
-											disabled={ loading }
-											compact
-										>
-											{ translate( 'Reset Password' ) }
-										</Button>
 									</>
 								) }
 							</td>
 						</tr>
+						{ ! disabled && (
```

Comment:
Is there any reason for hiding the password placeholder when disabled?

Reply:
Ahhh, I'll go ahead and add that in. Thanks!